src/components/Main/Login/Login.component.jsx

```diff
@@ -1,22 +1,61 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { NavLink } from 'react-router-dom';
 import Form from '../Form/Form.component';
 
 const Login = props => {
-  const state = {
+  const initLoginstate = {
     email: '',
     password: ''
   };
 
+  const [loginState, setState] = useState(initLoginstate);
+
+  const handleChange = e => {
+    const { id, value } = e.target;
+    setState(prevState => ({
+      ...prevState,
+      [id]: value
+    }));
+  };
+
+  const handleLogin = e => {
+    e.preventDefault();
+    const { email, password } = loginState;
+    if (!email || !password) {
+      alert('Пожалуйста, заполните все поля в форме');
```

Comment:
Let's discuss how you would implement i18n if requested.

Reply:
React context

Reply:
got this, thanks =)